app/code/Magento/Csp/Model/Mode/ConfigManager.php

```diff
@@ -45,17 +50,20 @@ class ConfigManager implements ModeConfigManagerInterface
 
     /**
      * @param ScopeConfigInterface $config
+     * @param ModeConfiguredFactory $modeConfiguredFactory
      * @param Store $store
      * @param State $state
      * @param Http|null $request
      */
     public function __construct(
         ScopeConfigInterface $config,
+        ModeConfiguredFactory $modeConfiguredFactory,
         Store $store,
         State $state,
         ?Http $request = null
     ) {
         $this->config = $config;
+        $this->modeConfiguredFactory = $modeConfiguredFactory;
```

Comment:
Could you please update new constructor parameter based on [Backward compatibility documentation](https://developer.adobe.com/commerce/contributor/guides/code-contributions/backward-compatibility-policy/#adding-a-constructor-parameter)